recipe/meta.yaml

```diff
@@ -20,7 +19,6 @@ requirements:
     - python
     - pip
     - flit-core >=2,<3
-    - setuptools
```

Comment:
why removing here setuptools?

Reply:
`flit-core` is a build backend https://github.com/tiangolo/typer/blob/d5bc56177edcb14f159743e5eaf52cd448e622dc/pyproject.toml#L3 and **no** setup.py file